### ISSUE TITLE
I get some problem when I do the task of cws with keras-bert 

### ISSUE THREAD

**Issue by kiko441500** [CyberZHG/keras-bert#151]
https://drive.google.com/open?id=19uxoBhWJeZVLRJsXLGeVVveg22-XLdP6
here is the file of ipynb. I can bulid model. I can also use the pre-trained model for the fine-tune task.
when I train the model. I got the accuracy 99%. And the model looks good. But when I do the predict task, I find the model works bad bad.
So I modify the following layers. First, I used crf which is import form keras.contrib. The accuracy is 35%. I donot konw what is the matter. 
If there is anyone can help me ? Thanks very much.
